src/vaadin-avatar.html

```diff
@@ -134,6 +150,14 @@
               observer: '__nameChanged'
             },
 
+            /**
+             * Color index used for avatar border.
+             */
+            colorIndex: {
+              type: Number,
+              observer: '__colorIndexChanged'
```

Comment:
nit: should it reflect to attribute? 🤔

Reply:
I think it's not needed for now. We can change it later if someone needs to style avatars based on the index.